src/components/notes/List/NoteRow.jsx

```diff
@@ -82,17 +83,22 @@ const NoteRow = ({ note, f, t, client }) => {
         >
           <Icon icon={NoteIcon} size={32} className="u-mr-1 u-flex-shrink-0" />
 
-          <div className="u-flex-grow-1">
-            <span className="u-charcoalGrey u-ellipsis">{filename}</span>
-
-            <WithBreakpoints hideOn={Breakpoints.Mobile}>
+          <WithBreakpoints hideOn={Breakpoints.Mobile}>
```

Comment:
Maybe you can use an inline If-Else with conditional operator like this
```jsx
{ isMobile ?  (
  <div
    className="u-flex u-flex-column u-flex-grow-1"
    style={{ minWidth: 0 }} >
    <span className="u-charcoalGrey u-ellipsis">{filename}</span>
     <NotePath drivePath={drivePath} path={note.path} noLink />
  </div>) : (
  <div className="u-flex-grow-1">
    <span className="u-charcoalGrey u-ellipsis">{filename}</span>
    <span className="u-ellipsis">{extension}</span>
  </div>)
}
```

Reply:
Easier to understand I agree 👍